content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -2,135 +2,131 @@
+
+   The chatter section is available on pricelist forms, which allows you to add
+   notes and communications on each pricelist page.
 
-   It should also be noted that, in Odoo 17 (and above), a chatter section can be found on pricelist
-   forms, which enhances the ability to communicate about them.
+Creating and editing pricelists
+-------------------------------
 
-From the :guilabel:`Pricelists` page, either select the desired pricelist to edit, or click
+From the :guilabel:`Pricelists` page, either select the pricelist to edit, or click
 :guilabel:`New` to create a new pricelist, which reveals a blank pricelist form that can be
 configured in a number of different ways.
 
-.. image:: pricing/pricelist-detail-form.png
-   :align: center
+.. image:: pricing/18-sales-new-pricelist-form.png
    :alt: How the pricelist detail form looks in Odoo Sales.
 
-When creating a new pricelist, start by adding a name for the pricelist at the top of the form, in
-the blank field. Next, select which :guilabel:`Currency` should be used.
+When creating a new pricelist, start by adding a name for the pricelist at the blank field at top of
+the form. Next, select which :guilabel:`Currency` should be used.
 
-Then, if working in a multi-company environment, select which company this pricelist should apply to
+If working in a multi-company environment, select which company this pricelist should apply to
 in the :guilabel:`Company` field. If this field is left blank, the pricelist is automatically
 applied to all companies in the database.
 
+If working in a multinational company, select the countries where this pricelist will apply under
+the :guilabel:`Country Groups` column.
+
 Price Rules tab
 ---------------
 
-The :guilabel:`Price Rules` tab functionality on a pricelist form varies depending on the
-:guilabel:`Pricelists` setting chosen: either :guilabel:`Multiple prices per product` or
-:guilabel:`Advanced price rules (discounts, formulas)`.
+In the :guilabel:`Price Rules` tab, each line creates a new record that will implement customized
+pricing to the sales order where the pricelist is applied. To create a new price rule, click on
+:guilabel:`Add a line`, which opens a new pricelist rules form.
 
-However, the :guilabel:`Time-based rules` tab and :guilabel:`Configuration` tab are always the same,
-regardless of the chosen :guilabel:`Pricelists` setting.
+Then, select whether to apply this set of rules to a :guilabel:`Product` or
+:guilabel:`Category`.
 
-Multiple prices per product
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+From here, there are several configuration options:
 
-With the :guilabel:`Multiple prices per product` setting activated, the :guilabel:`Price Rules` tab
-on pricelist forms provides the option to add specific products, with a specific price, to a
-pricelist.
+- :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one
+   or more products to which this pricelist will apply.
```

Comment:
Text in a list **must** align with the first character after the bullet and space. Please remove the extra space here :)
```suggestion
- :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one
  or more products to which this pricelist will apply.
```